packages/esm-patient-immunizations-app/src/immunizations/immunizations-overview.test.tsx

```diff
@@ -91,7 +91,7 @@ describe('ImmunizationOverview: ', () => {
       expect(screen.getByRole('columnheader', { name: new RegExp(header, 'i') })).toBeInTheDocument();
     });
 
-    const expectedTableRows = [/rotavirus sep-2018/, /polio nov-2018/, /influenza may-2018/];
+    const expectedTableRows = [/rotavirus sept — 2018/, /polio nov — 2018/, /influenza may — 2018/];
```

Comment:
I think this one should probably not be `wide`.

Reply:
Updated with 8ce9c4e